Mathlib/MeasureTheory/Integral/Average.lean

```diff
@@ -71,9 +71,28 @@ average w.r.t. the volume, one can omit `∂volume`. -/
 noncomputable def laverage (f : α → ℝ≥0∞) := ∫⁻ x, f x ∂(μ univ)⁻¹ • μ
 #align measure_theory.laverage MeasureTheory.laverage
 
-notation3"⨍⁻ "(...)", "r:60:(scoped f => f)" ∂"μ:70 => laverage μ r
+@[inherit_doc laverage] notation3"⨍⁻ "(...)", "r:60:(scoped f => f)" ∂"μ:70 => laverage μ r
+
+/-- Average value of an `ℝ≥0∞`-valued function `f` w.r.t. to the standard measure.
+
+It is defined as `volume univ⁻¹ * ∫⁻ x, f x`, so it is equal to zero if the space has infinite
```

Comment:
No, it is defined as `∫⁻ x, f x ∂(volume univ⁻¹ • volume)` so that it's an integral w.r.t. some measure.

Reply:
This came from the existing docstrings, which I have now modified too.